ydb/docs/en/core/getting_started/kubernetes.md

```diff
@@ -0,0 +1,325 @@
+
+Apply the manifest for creating a database and dynamic nodes:
+
+```bash
+kubectl apply -f samples/database.yaml
+```
+
+{% note info %}
+
+The value referenced by `.spec.storageClusterRef.name` key must match the name of the `Storage` resource with storage nodes.
+
+{% endnote %}
+
+A `StatefulSet` object that describes a set of dynamic nodes is created after processing the manifest. The created database will be accessible from inside the {{ k8s }} cluster by the `database-sample` hostname or the `database-sample.<namespace>.svc.cluster.local` FQDN, where `namespace` indicates the namespace that was used for the installation. You can connect the database via port 2135.
+
+View the status of the created resource:
+
+```bash
+kubectl describe database.ydb.tech
+```
+
+Result:
+
+```
+Name:         database-sample
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+API Version:  ydb.tech/v1alpha1
+Kind:         Database
+...
+Status:
+  State:  Ready
+Events:
+  Type     Reason              Age    From          Message
+  ----     ------              ----   ----          -------
+  Normal   Provisioning        8m10s  ydb-operator  Resource sync is in progress
+  Normal   Provisioning        8m9s   ydb-operator  Resource sync complete
+  Normal   TenantInitialized   8m9s   ydb-operator  Tenant /root/database-sample created
+```
+
+`State: Ready` means that the database is ready to be used.
+
+
+### Test cluster operation
+
+Check how {{ ydb-short-name }} works:
+
+
+ 1. Check that all nodes are in the `Running` status:
+
+    ```bash
+    kubectl get pods
+    ```
+
+    Result:
+
+    ```
+    NAME                READY   STATUS    RESTARTS   AGE
+    database-sample-0   1/1     Running   0          1m
+    database-sample-1   1/1     Running   0          1m
+    database-sample-2   1/1     Running   0          1m
+    database-sample-3   1/1     Running   0          1m
+    database-sample-4   1/1     Running   0          1m
+    database-sample-5   1/1     Running   0          1m
+    storage-sample-0    1/1     Running   0          1m
+    storage-sample-1    1/1     Running   0          1m
+    storage-sample-2    1/1     Running   0          1m
+    storage-sample-3    1/1     Running   0          1m
+    storage-sample-4    1/1     Running   0          1m
+    storage-sample-5    1/1     Running   0          1m
+    storage-sample-6    1/1     Running   0          1m
+    storage-sample-7    1/1     Running   0          1m
+    storage-sample-8    1/1     Running   0          1m
+    ```
+
+ 2. Start a new pod with [{{ ydb-short-name }} CLI]((../reference/ydb-cli/index.md)):
+
+    ```bash
+    kubectl run -it --image=cr.yandex/yc/ydb:21.4.30 --rm ydb-cli bash
```

Comment:
Do we still want the 21.4.30 version as an example? I feel like both this place and the sample in ydb-kubernetes-operator should be updated to something a bit newer. At the moment, samples in operator come with 22.4.44, maybe we should replace the version here with 22.4.44 at least as well? 

Do not forget about an equivalent place in Russian docs

Reply:
UPD it turned out that `cr.yandex/yc/ydb` and `cr.yandex/crptqonuodf51kdj7a7d/ydb` are not quite the same. We simply cannot replace `21.4.30 -> 22.4.44` and expect everything to work. We will also need to replace `cr.yandex/yc` with `cr.yandex/crptqonuodf51kdj7a7d`, because we don't have direct access to `cr.yandex/yc` and don't actually push there that often.

Given that we will be trying to move to Dockerhub in not so distant future anyway, I suggest replacing `cr.yandex/yc/ydb:21.4.30` with `cr.yandex/crptqonuodf51kdj7a7d/ydb:22.4.44` and call it a day for now. What do you think @blinkov?

Reply:
```suggestion
    kubectl run -it --image=cr.yandex/crptqonuodf51kdj7a7d/ydb:22.4.44 --rm ydb-cli bash
```

Reply:
done